synology_dsm/synology_dsm.py

```diff
@@ -351,6 +361,12 @@ def reset(self, api):
             if api == SynoSurveillanceStation.API_KEY:
                 self._surveillance = None
                 return True
+            if api == SynoCoreSystem.API_KEY:
+                self._system = None
+                return True
+            if api == SynoCoreUpgrade.API_BASE_KEY:
+                self._upgrade = None
+                return True

```

Comment:
Order them with alphabetically

(Fix `SynoDownloadStation` order btw)

Reply:
order is fixed on many places :)